### DISCUSSION TITLE
添加域名分流经常失败

### DISCUSSION THREAD

**Discussion by FelixxChoi** [joyanhui/ikuai-bypass#38] [General]
执行任务，前面都很正常，到域名分流的时候开始经常性失败，如以下：

...............2024/11/19 13:33:48 域名分流=   1 / 1 wan2 wan2 重试失败，可能是列表太多了，添加太快,爱快没响应。已经重试过一次，所以跳过此次操作
2024/11/19 13:33:48 域名分流== 添加域名分流 'wan2' 失败：Post "http://10.1.1.1/Action/call": context deadline exceeded (Client.Timeout exceeded while awaiting headers)

想问下是先删掉旧的列表再更新，还是先更新后删除旧的列表啊？

**Comment by joyanhui** [comment]
域名分流规则  有一个参数 是 -delOldRule 默认是  after 也就是先添加成功后再删除。你可以添加启动参数    -delOldRule  before  改为先删除后添加。


爱快无响应的问题，困扰很久，经过几个用户的反复测试 目前怀疑是爱快硬件设备性能太低导致的（主要是cpu单核性能太烂）或者是爱快在低性能设备上限制了api的资源。暂时没有好的解决方案。一个折中的方案是用ikuai-bypass的导出域名列表功能，然后再爱快的网页端手动导入到爱快内


命令参考 
```sh
./ikuai-bypass -exportPath /tmp/domainlist.txt   -r exportDomainSteamToTxt
```
